Source/WebKit/UIProcess/mac/WKFullScreenWindowController.mm

```diff
@@ -741,7 +744,10 @@ - (void)_continueExitingFullscreenAfterPostingNotificationAndExitImmediately:(bo
     }
 
     page->updateRenderingWithForcedRepaint([weakSelf = WeakObjCPtr<WKFullScreenWindowController>(self)] {
-        [weakSelf completeFinishExitFullScreenAnimation];
+        RetainPtr protectedSelf = weakSelf.get();
```

Comment:
I think we use `strongSelf` naming in ObjC. Also, I usually go for the more concise form when there is a single line like this:
```
if (RetainPtr strongSelf = weakSelf.get())
    [strongSelf completeFinishExitFullScreenAnimation];
```